### DISCUSSION TITLE
More props in the config.json files of themes

### DISCUSSION THREAD

**Discussion by ebattistini** [OnionUI/Onion#1498] [Ideas]
I've made my first theme recently and started playing a bit more with the theming capabilities. 
Would it be possible to have a drop shadow bool or colours values prop for the text shadow ? On dark theme it looks ok but when working on light themes it makes the text appear double

Unless there is a property I have missed somewhere ?

![MainUI_001](https://github.com/OnionUI/Onion/assets/6288298/46e12862-5fc7-4de9-9abb-d8c60c96c41f)

**Comment by Aemiii91** [comment]
Yes, there will be in the future - when we make our own launcher (the current launcher is closed source)

**Comment by thomasklemenc** [comment]
I have a PR up to disable them until we release our own Launcher #1427

**Reply by ebattistini** [reply]
wow! that's the fastest comments I had in a while! Nice one. I've been really enjoying the community and if you ever need UX/Ui help feel free to ping 😉

**Comment by ebattistini** [comment]
Is there a design project being considered and some designers interested ? If we are planing a launcher build one day we should def get some designers together to think about it and mock some concepts?
